### DISCUSSION TITLE
I cant create an account

### DISCUSSION THREAD

**Discussion by ZSolarDev** [ppy/osu#27118] [Q&A] [answered]
For some reason, when I click the Register button to create an account, I get this error saying, "Something happened... but we're not sure what."
![Screenshot 2024-02-10 124452](https://github.com/ppy/osu/assets/101154324/aaf0afd8-f550-4ba7-b999-3aac1f61450e)
Not only that, but everytime I click the button, these 2 lines keep getting added to the network log,
```
2024-02-10 17:44:30 [verbose]: Request to https://lazer.ppy.sh/users failed with System.Net.WebException: Forbidden.
2024-02-10 17:44:30 [verbose]: Response was: {"error":"disabled"}
```
Please help!

**Comment by bdach** [comment, marked as answer]
Registration was temporarily disabled at the time due to some people doing stuff they really shouldn't be. Please try again.

**Reply by Elmo132978** [reply]
it happened to me too

**Reply by bdach** [reply]
@Elmo132978 please attach logs, button to export is in settings.

**Reply by Alejo1093** [reply]
I have the same issue

**Reply by peppy** [reply]
@Alejo1093 did you figure it out?

**Reply by Hurenocek** [reply]
[compressed-logs.zip](https://github.com/user-attachments/files/27373834/compressed-logs.zip)
Проблема та же
